### PR TITLE
lakefs fs spark test app

### PR DESCRIPTION
**Pull request by nopcoder**
Spark application for testing lakeFS filesystem.
The app process around ~10GB from [Amazon Reviews PDS](https://s3.amazonaws.com/amazon-reviews-pds/readme.html), partition the data by year, write to lakeFS, load the result and dump top 10.

### CODE REVIEW

File: test/lakefsfs/build.sbt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name := "lakefsfs"
+
+version := "0.1"
+
+scalaVersion := "2.12.10"
+
+libraryDependencies ++= Seq(
+    "org.apache.spark" %% "spark-sql" % "3.1.1" % "provided",
```
Comment (arielshaqed) [inline test/lakefsfs/build.sbt:8]: Do we want to go Spark 3.1.1?  I guess we'll see when we ask customers to run it :-)

**Review by arielshaqed — APPROVED** [review]
Neat!

File: test/lakefsfs/build.sbt (hunk 1 above)
Comment (nopcoder) [reply test/lakefsfs/build.sbt:8]: Had to run it locally
But I'll check if there is a common way to generate a multi-target solution.

Comment (nopcoder) [reply test/lakefsfs/build.sbt]: @arielshaqed updated the build to enable the above.

**Pull request merged**